bagel/pheno_utils.py

```diff
@@ -35,6 +39,30 @@ def validate_portal_uri(portal: str) -> Optional[str]:
     return portal
 
 
+def load_pheno(input_p: Path) -> pd.DataFrame | None:
+    """Load a .tsv pheno file and do some basic validation."""
+    if input_p.suffix != ".csv":
+        pheno_df: DataFrame = pd.read_csv(
+            input_p, sep="\t", keep_default_na=False, dtype=str
+        )
+
+        if pheno_df.shape[1] > 1:
+            # We only extracted one column, this might mean that we have a sneaky .csv
+            return pheno_df
+        elif len(pheno_df.columns[0].split(",")) == 1:
```

Comment:
Good point, I agree this is confusing. Let me take another look with the broader file type filtering (i.e. `"Only .tsv"` instead of `"not .csv" and "not .txt"`). Maybe that'll help

Reply:
Having looked at this again, I think the implementation was just wrong here. What I wanted to see was:

- do you pretend to be a tsv
- and have only one column
- but if I split your header by comma, I get more than one element

-> congrats, you might suffer from undeclared .csv-ness. 
My check was something different